src/replit_river/client_session.py

```diff
@@ -45,6 +43,9 @@
 logger = logging.getLogger(__name__)
 
 
+STREAM_CLOSED_BIT = 0x0004  # Synonymous with the cancel bit in v2
```

Comment:
Can we use the other format for consistency

Reply:
Yep!